Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -283,6 +283,7 @@ public struct ConvertAction: Action, RecreatingContext {
     /// Converts each eligible file from the source documentation bundle,
     /// saves the results in the given output alongside the template files.
     mutating public func perform(logHandle: LogHandle) throws -> ActionResult {
+        let totalTimeMetric = benchmark(begin: Benchmark.Duration(id: "convert-total-time"))
```

Comment:
Should we just replace the existing `convert-action` metric with this one? I think it's a little confusing to understand the difference between them now.

Reply:
I agree that it's confusing but changing the meaning of a metric means that values before that change can't be compared with values after that change. I think the best way to do this would be to remove the old metric and use a different name for the new metric. That way they won't accidentally be compared to each other.

Reply:
That sounds good to me 👍🏻

Reply:
Adressed in d5001f0978589e17c04e6c41d8c64af72a8d0acd